airflow/config_templates/config.yml

```diff
@@ -1111,6 +1111,13 @@ metrics:
       type: string
       example: ~
       default: "localhost"
+    statsd_ipv6:
+      description: |
+        Enables the statsd host to be resolved into IPv6 address
+      version_added: 2.10.3
```

Comment:
That will be 3.0.0. We currently stopped adding new features to Airflow 2 and any new features go by default to 3.0.0.

Reply:
At least - that is the default. If there is a good reason to make it, that would still be 2.11.0 not 2.10.3 (2.10.3 is a bugfix release). 

If you would like to make it into any of the 2.* releases, the right place is to start a discussion about it in `#release-management' channel in Slack or at the devlist.

Reply:
I am fine with put it in 3.0.0.